h11/_writers.py

```diff
@@ -7,14 +7,17 @@
 # - a writer
 # - or, for body writers, a dict of framin-dependent writer factories
 
-from ._events import Data, EndOfMessage
+from typing import Callable, Dict, List, Tuple, Type, Union
+
+from ._events import Data, EndOfMessage, Event, InformationalResponse, Request, Response
+from ._headers import Headers
 from ._state import CLIENT, IDLE, SEND_BODY, SEND_RESPONSE, SERVER
-from ._util import LocalProtocolError
+from ._util import LocalProtocolError, Sentinel
 
 __all__ = ["WRITERS"]
 
 
-def write_headers(headers, write):
+def write_headers(headers: Headers, write: Callable[[bytes], None]) -> None:
```

Comment:
Since we don't care about the return type of `write` should be `Callable[[bytes], Any]`?

Reply:
Yea, changed. Thanks